packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -31,10 +32,23 @@ const FinishSocialLoginPage = lazy(
 );
 
 const NotificationsPage = lazy(() => import('views/pages/notifications'));
-const NotificationSettingsPage = lazy(
-  () => import('views/pages/notification_settings'),
+
+const NotificationSettingsOld = lazy(
+  () => import('views/pages/NotificationSettingsOld'),
+);
+const NotificationSettings = lazy(
+  () => import('views/pages/NotificationSettings'),
 );
 
+const NotificationSettingsPage = () => {
+  const knockInAppNotifications = useFlag('knockInAppNotifications');
+  if (knockInAppNotifications) {
+    return <NotificationSettings />;
+  } else {
+    return <NotificationSettingsOld />;
+  }
+};
+
```

Comment:
features flags like `knockInAppNotifications`, should come from Router.tsx per existing convention. ex:

https://github.com/hicommonwealth/commonwealth/blob/9885315df5941a3a57c82abccb911f4471ad587d/packages/commonwealth/client/scripts/navigation/Router.tsx#L31-L35

Reply:
please also update this in `CustomDomainRoutes.tsx`